### PR TITLE
Paraview 5.12 prep

### PR DESCRIPTION
**Pull request by mathstuf**

### CODE REVIEW

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -226,7 +226,8 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("protobuf@3.4:3.18", when="@:5.10%xl_r")
     # protobuf requires newer abseil-cpp, which in turn requires C++14,
     # but paraview uses C++11 by default
-    depends_on("protobuf@3.4:3.21", when="@:5.11")
+    depends_on("protobuf@3.4:3.21", when="@:5.12")
```
Comment (kwryankrattiger) [inline var/spack/repos/builtin/packages/paraview/package.py]: I don't understand this change. I think if this is going to be true it seems always, then maybe we can just drop the version constraint?

Comment (mathstuf) [reply var/spack/repos/builtin/packages/paraview/package.py]: I suppose we can add a comment to add an upper build when we finally use something newer than C++11.

**Comment by Billae** [comment]
I would really appreciate to see this merge in the master branch,  I tried to install paraview today, forgetting to checkout this branch and it still failed (which is totally normal ^^ ).
Regards,
Eloïse

File: var/spack/repos/builtin/packages/paraview/package.py (hunk 1 above)
Comment (mathstuf) [reply var/spack/repos/builtin/packages/paraview/package.py]: I made it for 5.11+ and added a note to check it when ParaView is no longer C++11.

**Review by kwryankrattiger — APPROVED** [review]
LGTM

**Pull request merged**